addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -202,11 +200,7 @@ html[data-edit_translations="1"] {
 
     // CC Edition
     .o_we_cc_preview_wrapper {
-        // Use box-shadow rather than border-bottom in order to
-        // avoid misalignments in the 'Options' tab.
-        border: 1px solid;
-        border-color: rgba($o-we-item-standup-color-light, .2) $o-we-sidebar-content-field-dropdown-border-color transparent;
-        box-shadow: 0 1px 0 $o-we-item-standup-color-dark;
+        border: $o-we-dropdown-border-width solid $o-we-sidebar-content-field-dropdown-border-color;
```

Comment:
Probably this change that creates this bug:
![image](https://user-images.githubusercontent.com/10338094/234645304-6883ce3c-6e6e-4bd4-88f5-295275bd2f21.png)